.drone.yml

```diff
@@ -0,0 +1,34 @@
+---
+kind: pipeline
+name: default
+
+x-settings: &dockersettings
+  username:
+    from_secret: docker_username
+  password:
+    from_secret: docker_password
+  repo: livingdocs/varnish
+  tags: [5.2.1-r5, latest]
+
+steps:
+- name: build
+  image: plugins/docker
+  settings:
+    dry_run: true
+    <<: *dockersettings
+  when:
+    event: [push]
+
+- name: publish
+  image: plugins/docker
+  settings:
+    <<: *dockersettings
+  when:
+    branch: [master]
+    event: [push]
```

Comment:
DroneCI v1 has pull request builds always enabled.
Without this event condition, it would always trigger the master build.
We have to figure out how to properly configure steps or pipelines.